sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/Samples/Sample04_HookCrudOperations.cs

```diff
@@ -73,7 +72,7 @@ public async Task GetHookAsync()
                 Console.WriteLine($" - {admin}");
             }
 
-            if (hook.HookType == HookType.Email)
```

Comment:
to make sure I understand, so this property was internal and hidden from the user. And with this PR we are renaming it and exposing it to the user. right?
So was this a bug in the sample code? or did I miss something

Reply:
> to make sure I understand, so this property was internal and hidden from the user. And with this PR we are renaming it and exposing it to the user. right?

Correct.

>  So was this a bug in the sample code? or did I miss something

Yes. We were accessing an internal property by accident. We do have access to them because we are exposing internals to the test project. I wonder if we should split samples into another project to prevent this from happening again.